REST-API-ADVANCED/advanced/src/main/java/com/epam/esm/security/feign/dto/TokenVerifiedDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.esm.security.feign.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class TokenVerifiedDTO {
+    private String iss;
+    private String azp;
+    private String email;
+    private Boolean email_verified;
+    private String at_hash;
+    private String name;
```

Comment:
Please use java code convention names. If you need this name to get the value from json response, there is specific annotations for this in each json mapper.

Reply:
done